client/README.md

```diff
@@ -1,7 +1,7 @@
 # Cribbage Frontend
-This is the React frontend for the cribbage application. To run:
+This is the React frontend for the cribbage application. To run it, navigate to the root directory and run:
 ```
-npm install
-npm run dev
+make client
 ```
-The `dev` script will spin up the Go server (so make sure you can run that first) with an in-memory db (for now) as well as the React app, which will open in a browser.
+
+To run against the local server, be sure you are also running `make serve`
```

Comment:
Should we just instruct people to run `make reactdev`?

Reply:
Wait... did you get rid of `reactdev`?

Reply:
yap. 🤞 that `make client` talks to the local gin server just fine.

Reply:
Okay so the assumption is that when developing react stuff you can just start the server once and restart the client as needed, therefore making a single command to do both not super useful?

Reply:
In that case we'll want to do an `npm uninstall concurrently` since it's no longer a dev dependency